flaps/flaps.go

```diff
@@ -163,23 +162,32 @@ func (f *Client) CreateApp(ctx context.Context, name string, org string) (err er
 		"org_slug": org,
 	}
 
-	_, err = f._sendRequest(ctx, http.MethodPost, "/apps", in, nil, nil)
+	err = f._sendRequest(ctx, appCreate, http.MethodPost, "/apps", in, nil, nil)
 	return
 }
 
-func (f *Client) _sendRequest(ctx context.Context, method, endpoint string, in, out interface{}, headers map[string][]string) (int, error) {
+func (f *Client) _sendRequest(ctx context.Context, action flapsAction, method, endpoint string, in, out interface{}, headers map[string][]string) error {
+	ctx, span := tracing.GetTracer().Start(ctx, fmt.Sprintf("flaps.%s", action.String()), trace.WithAttributes(
+		attribute.String("request.action", action.String()),
+		attribute.String("request.endpoint", endpoint),
+		attribute.String("request.method", method),
```

Comment:
some requests have specific headers that are pretty useful context for what the request is doing. Should we include that?